setup.sh

```diff
@@ -25,13 +25,18 @@ fi
 export NUM_AVAILABLE_IPU=${IPU_ARG}
 export GRAPHCORE_POD_TYPE="pod${IPU_ARG}"
 
+# Logger specific vars
+export TIER_TYPE=$(python .gradient/check_tier.py)
+export FIREHOSE_STREAM_NAME="paperspacenotebook_production"
+
```

Comment:
Maybe group the three variables? It'll make it easier to port to other repos

Reply:
What do you mean?

Reply:
Ah right, I didnt see the third one down below, thanks!

Reply:
I changed it